.github/workflows/brew-test.yml

```diff
@@ -1,16 +0,0 @@
-name: homebrew-test
```

Comment:
Removed. This cant be helped in this repository anyway and is part of https://github.com/returntocorp/homebrew-semgrep CI. Will make that GHA run on cron.